### PR TITLE
pablo/ci-cd-openshift

### PR DESCRIPTION
**Pull request by PabloGalegoC**
Setting up openshift infrastructure and a CI-CD pipeline for [spring-boot-maven-angular-starter](https://github.com/PabloGalegoC/spring-boot-maven-angular-starter)

### CODE REVIEW

File: Pablo/ngboot-app/doc/doc-daily-minishift.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#Minishift Documentation
+
+##Minishift installation
```
Comment (franrua) [inline Pablo/ngboot-app/doc/doc-daily-minishift.md]: In markdown headers must be space between `#` and the text of header. Apply markdwn conventions

File: Pablo/ngboot-app/doc/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Runing the demo
+
+At the project root directory
+
+`. Pablo/ngboot-app/openshift/ops/provision deploy`
+Dafault username "pablo" could be changed adding `--user <user>`
```
Comment (franrua) [inline Pablo/ngboot-app/doc/README.md]: Strange line break. You must set up and start the minishift before anything.

File: Pablo/ngboot-app/doc/doc-daily-minishift.md (hunk 1 above)
Comment (PabloGalegoC) [reply Pablo/ngboot-app/doc/doc-daily-minishift.md]: Done

File: Pablo/ngboot-app/doc/README.md (hunk 2 above)
Comment (PabloGalegoC) [reply Pablo/ngboot-app/doc/README.md]: Done

**Comment by franrua** [comment]
Cerrado porque el responsable cometió alta traición

**Pull request closed without merging**